### PR TITLE
CT-1162: Update copy from PCPro to Rev Gen

### PR DESCRIPTION
**Pull request by tomasstark**
This PR addresses [CT-1162](https://laterpay.atlassian.net/browse/CT-1162).

* It updates the plugin name in user-facing parts of the plugin to "Revenue Generator".

### CODE REVIEW

File: onboarding/templates/welcome-existing.php [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?php
 /**
- * Paid Content Pro admin welcome screen for Revenue Generator merchants.
+ * Revenue Generator admin welcome screen for Revenue Generator merchants.
  *
  * @package paid-content-pro
```
Comment (leesh22) [inline onboarding/templates/welcome-existing.php:5]: Do you need to change this, or will that be a diff PR? I know this one is user facing.

File: onboarding/templates/welcome-new.php [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <?php
 /**
- * Paid Content Pro admin welcome screen.
+ * Revenue Generator admin welcome screen.
  *
  * @package paid-content-pro
```
Comment (leesh22) [inline onboarding/templates/welcome-new.php:5]: Again here

**Review by leesh22 — APPROVED** [review]

File: onboarding/templates/welcome-existing.php (hunk 1 above)
Comment (tomasstark) [reply onboarding/templates/welcome-existing.php:5]: Good point. Yes, it's good that we change `@package` in this PR. On it 👀

Comment (tomasstark) [reply onboarding/templates/welcome-existing.php]: Fixed in https://github.com/laterpay/revenue-generator/pull/300/commits/36f4d4878d983ab83861f8ad612cadf02f9c1af0.

File: onboarding/templates/welcome-new.php (hunk 2 above)
Comment (tomasstark) [reply onboarding/templates/welcome-new.php]: Fixed in https://github.com/laterpay/revenue-generator/pull/300/commits/36f4d4878d983ab83861f8ad612cadf02f9c1af0.

**Pull request merged**